src/Pages/ara/Main.js/Feed.js

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import "./Feed.scss";
+import Comment from "./Comment.js";
```

Comment:
```suggestion
import React from "react";
import Comment from "./Comment.js";
import "./Feed.scss";
```
어디에 뭐가 있는지 손만 가도 알 수 있도록 서랍의 순서를 정리하듯이 import 순서도 잘 정리해주셔야 합니다. 별거 아닐 수 있는 부분 처럼 보일 수 있겠으나 이런 약속들이 모여 개발의 생산성을 높여줍니다.